Bio/Align/Applications/_Muscle.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, cmd="muscle", **kwargs):
             ["pctid_kimura", "pctid_log"]
         OBJECTIVE_SCORES = ["sp", "ps", "dp", "xp", "spf", "spm"]
         TREE_ROOT_METHODS = ["pseudo", "midlongestspan", "minavgleafdist"]
-        SEQUENCE_TYPES = ["protein", "nucleo", "auto"]
+        SEQUENCE_TYPES = ["protein", "rna", "dna", "nucleo", "auto"]
```

Comment:
Can you add a comment line above this about the historic versus current usage?
If we know which version of MUSCLE this change happened in, even better.

Reply:
Hi Peter, I'll add a comment regarding this. I'm going send an email to the MUSCLE author as version history is not available and I can't pin-point when this change occurred. We know the current version 3.8.31 uses the new seqtype arguments, while the 3.8 user manual lists the old arguments (which could possibly be an error).